src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentContents.js

```diff
@@ -31,6 +31,16 @@ export class DocumentContents extends Component {
             </>
           )}
 
+          {!_isEmpty(document.metadata.physical_description) && (
```

Comment:
I think you need to add the `physical_description` in this `if` too 
https://github.com/inveniosoftware/react-invenio-app-ils/blob/5669e2467284fbdb5ef99209e7a50c710be93963/src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentContents.js#L15 because otherwise if all the other metadata are empty except the `physical_description`, the content tab will be empty.

Reply:
Nice catch, thanks!